samples/app-hello-world/nodejs/README.md

```diff
@@ -164,6 +170,26 @@ The simplest way to run this sample in Teams is to use Teams Toolkit for Visual
 
 **Note:** Similarly, you can test your application in the Office 365 desktop app as well.
 
+**Element RealtionShip Property**
+**Oneway Dependency - Bot and Message Extension both should work**
+![BotResponse](Images/ElementRelationShip/3.ResponseFromBot_OnewayDepency.png)
+
+![MessageExtension](Images/ElementRelationShip/4.MsgExt_OneWayDependency.png)
+
+**Mutual Dependency - Bot,Tab and Configurable Tab all three should work**
+**Group Scope**
+![ConfigurableTab](Images/ElementRelationShip/6.ConfigureTab.png)
+
+![GroupChatTab](Images/ElementRelationShip/7.GC_Tab_MutualDependency.png)
+
+![GroupChatBot](Images/ElementRelationShip/8.GC_Bot_MutualDependency.png)
+
+![GroupChatMessageExt](Images/ElementRelationShip/9.GC_MsgExtension.png)
+
+**Outlook - Oneway Dependency**
+**Outlook doesn't support bots, and since Message Extensions depend on a bot, the Message Extension doesn't work in Outlook**
```

Comment:
This is wrong in two aspects:
1. If RT is used correctly to specify actionME dependsOn bot, then the actionME shouldn't even show up in Outlook. But in your illustration, the command shows up.
2. The fact that the command shows up as your illustration displays is actually a bug - the command shows up yet doesn't work. It doesn't represent the correct end result of using RT.